multi_test.gemspec

```diff
@@ -3,7 +3,7 @@ $LOAD_PATH.unshift File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = 'multi_test'
-  s.version     = '0.1.2'
+  s.version     = File.read(File.join(File.expand_path(File.dirname(__FILE__)),'VERSION'))
```

Comment:
File.dirname(__FILE__) has a shortcut from Ruby 2.

__dir__

Hope this helps!

Reply:
Ger markdown I'll edit my underscores.

Reply:
There, sorry for the noise.